src/actions/promote.ts

```diff
@@ -146,42 +142,33 @@ async function promoteBlueGreen(
       'Routing to new deployments and deleting old workloads and services'
    )
    if (routeStrategy == RouteStrategy.INGRESS) {
-      await routeBlueGreenIngress(
+      await routeBlueGreenIngressUnchanged(
          kubectl,
-         null,
          manifestObjects.serviceNameMap,
          manifestObjects.ingressEntityList
       )
-      await deleteWorkloadsAndServicesWithLabel(
+
+      await deleteGreenObjects(
          kubectl,
-         GREEN_LABEL_VALUE,
-         manifestObjects.deploymentEntityList,
-         manifestObjects.serviceEntityList
+         manifestObjects.deploymentEntityList.concat(
```

Comment:
a popular pattern is to swap this to 
```javascript
[].concat(manifestObjects.deploymentEntityList, manifestObjects.serviceEntityList)
```
it's a little more readable about what's happening. It looks like you're calling a method on `manifestObjects.deploymentEntityList` in the original which implies it's more important than `manifestObjects.serviceEntityList`. Same functionality but doing it on an empty list shows the intent that you're merging them a bit better